### ISSUE TITLE
AlphaFold conversion to BinaryCIF fails

### ISSUE THREAD

**Issue by fkaiserbio** [rcsb/ciftools-java#10]
Hi!

Using the library in version 3.0.0 from Maven Central
```xml
<dependency>
    <groupId>org.rcsb</groupId>
    <artifactId>ciftools-java</artifactId>
    <version>3.0.0</version>
</dependency>
```
conversion to BinaryCIF fails with the example code given in the README. The file is not actually converted and contains mmCIF content. Please review the files attached. The gene name of this particular structure on UniProt is strange and contains a two quote characters: `''cytochrome P450`.

Code used for conversion:

```java
CifFile cifFile = CifIO.readFromPath(cifFilePath);
MmCifFile mmCifFile = cifFile.as(StandardSchemata.MMCIF);

// convert to BinaryCIF representation
byte[] output = CifIO.writeBinary(mmCifFile);
```

Thank you for checking.
[AF-O49373-F1-model_v1.zip](https://github.com/rcsb/ciftools-java/files/7569012/AF-O49373-F1-model_v1.zip)

**Comment by JonStargaryen** [comment]
Thanks for reporting @fkaiserbio. This should be fixed with 3.0.1.
Please let me know if you still see issues when converting files.

**Comment by fkaiserbio** [comment]
Hi @JonStargaryen,

thanks a lot for the quick fix. I can confirm that it works now for this structure. I will close the issue for now and let you know if there are problems with any other of the AlphaFold structures.

Best :wave:
